artemis-distribution/src/main/resources/licenses/bin/LICENSE

```diff
@@ -432,4 +432,11 @@ For Underscore:
 ==============================================================================
 This product bundles Underscore, which is available under a
 "MIT" license.  For details, see
-https://github.com/jashkenas/underscore/blob/master/LICENSE
\ No newline at end of file
+https://github.com/jashkenas/underscore/blob/master/LICENSE
+
+==============================================================================
+For Jakarta EE 8:
+==============================================================================
+This product bundles Jakarta EE 8, which is available under a
+"EPL 2.0" license.  For details, see
+https://github.com/eclipse-ee4j/jakartaee-api/blob/8.0.0/LICENSE.txt
```

Comment:
The actual licence needs to be in this file, or linked from it to another file elsewhere in the distribution. Links to external sites arent sufficient as they can come and go or change (say if the entire API stack moves to a different organisation)

(Yes this problem applies to the one right above it, which is also wrong, and further doesnt even link to a version as you try to)

We dont really use "jakartaee-api"/"Jakarta EE 8" either, only specific APIs, they should each be specifically called out that this applies to them. If the individual licence text of multiple items matches they can reuse the same definition.

Reply:
I have just pushed a commit to add a `licenses` folder with the required files and to specify the Jakarta EE 8 APIs bundled.